multi-storage-transaction-sample/README.md

```diff
@@ -183,62 +139,123 @@ After the initial data has loaded, the following records should be stored in the
 | 4       | Mango  | 5000  |
 | 5       | Melon  | 3000  |
 
-## Run the sample application
+## Execute transactions and retrieve data in the sample application
+
+The following sections describe how to execute transactions and retrieve data in the sample e-commerce application.
 
-Let's start with getting information about the customer whose ID is `1`:
+### Get customer information
 
-```shell
+Start with getting information about the customer whose ID is `1` by running the following command:
+
+```console
 $ ./gradlew run --args="GetCustomerInfo 1"
+```
+
+You should see the following output:
+
+```console
 ...
 {"id": 1, "name": "Yamada Taro", "credit_limit": 10000, "credit_total": 0}
 ...
 ```
 
-Then, place an order for three apples and two oranges by using customer ID `1`.
-Note that the order format is `<Item ID>:<Count>,<Item ID>:<Count>,...`:
+### Place an order
+
+Then, have customer ID `1` place an order for three apples and two oranges by running the following command:
+lol
+{% capture notice--info %}
```

Comment:
```suggestion

{% capture notice--info %}
```

Reply:
`lol`, thank you for catching that! I forgot I had that as a placeholder.😅